hydra/app/views/layouts/collection.html.erb

```diff
@@ -56,6 +56,13 @@
             <div class="alert alert-<%= key %>"><%= value %></div>
         <% end %>
         <div class="row">
+          <% if content_for?(:page_header) %>
```

Comment:
Would it make more sense to add this as a conditional in Bulkrax rather than here?